### DISCUSSION TITLE
How long, once an account has been deleted, does it take for the email to be freed up?

### DISCUSSION THREAD

**Discussion by Seadogpreedy** [community/community#147868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

HI can anyone help as I've used an email address in one Github(1) account which was set up in error and I then have another Github(2) account with a different primary email address; that now needs the email address associated with Github(1) account to authorise a User Package. What I need to know is if I delete the Github(1) account will that email associated with it be available to be used on GitHub(2) account straight away or is there a delay in availability? And if so how long that would be?
Cheers
Mark

**Comment by AviRahimov** [comment]
Try to do the following steps:

1) Change the Email on GitHub 1: Before deleting your first GitHub account, update its email to a different address. This action immediately frees up your desired email for reuse.

2) Add the Email to GitHub 2: Once the email is freed, add it to your second GitHub account. This process is instantaneous, allowing you to use the email without delay.

3) Delete GitHub 1 (Optional, only if you want): After transferring the email, you can delete the first GitHub account if it's no longer needed. Be aware that GitHub retains usernames for 90 days post-deletion to prevent reuse during this period. 
Another issue similar to this has already been opened. You might want to check it out here: [GitHub Community Discussion](https://github.com/orgs/community/discussions/125803)

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Seadogpreedy!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Profile category is solely related to conversations around the [GitHub product Profile](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Comment by arvindsunda6** [comment]
My account change
